tests/resource_test.js

```diff
@@ -0,0 +1,65 @@
+/// <reference path="./steps.d.ts" />
+Feature('Resource test');
+
+var FirstLogin = false;
+var loginCookies = [];
+const pdfName = `Dummy PDF ${Math.random() * 1000}`;
+const collectionName = `Dummy Collection ${Math.random() * 1000}`;
+
+Before((I) => {
+    I.login('admin', 'password');
+});
+
+Scenario('test upload a PDF file and verify the upload process', (I) => {
+    I.seeInCurrentUrl('#dashboard');
+    I.click('Library'); I.wait(3);
+    I.seeInCurrentUrl('#resources');
+
+    I.click('Add New Resource'); I.wait(3);
```

Comment:
There is a built in implicit wait in codeceptjs, so make sure you only add waits when needed and not just in case. As the number of tests increases each extraneous second turns into minutes. Its better to wait explicitly for what you need to appear, so it can handle slight variations in load time etc. (example: [waitForVisible](http://codecept.io/helpers/WebDriverIO/#waitforvisible)) This applies to pretty much all waits except waiting for dialogs I think, since I haven't found any wait for dialog yet.